content/docs/technical_practices/azure/azure_make_vm_with_cli.md

```diff
@@ -0,0 +1,74 @@
+---
+# Page title as it appears in the navigation menu
+title: "Quickstart: Make a new VM using the command line"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 1
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Azure via CLI basic quickstart
+
+## Goal
+
+Set up a basic Azure VM running Ubuntu using the command line tools.
+
+## Assumptions
+
+- You have the command line tools installed already.
+- You can recognise your subscription ID.
+- You know which region you want and which VM you want.
+
+## Steps
+
+1. `az login`, then find your subscription.
+1. Maybe you need `az account list --output table` to find your subscriptions,
+   and `az account set -s <subscription-id>` to set the correct one as the
+   default.
+1. `az group create --name <group-name> --location uksouth` to make a “resource
+   group” which holds all the various bits and pieces of this server. To get a
+   list of locations do `az account list-locations`.
+1. Make a VM, attach some storage units. The "Standard_B2s" size is a 2 core, 4
+   GB ram machine, with 30GB of local storage, which is suitable for "bursty"
+   tasks. (Basically, when you let it run at less than 100% for a while, you
+   build up 'credits' that let you run it at more than 100% for a time. Or
+   something like that.)
+
+   I've added a 128 GB disk. The "Standard_LRS" disk is an HDD, not an SSD.
```

Comment:
We could be clearer that the 128GB is an additional data disk (the VM will already come with an OS disk of some kind and size).

```suggestion
   I've added a 128 GB data-disk. The "Standard_LRS" disk is an HDD, not an SSD.
```